wandb/sdk/internal/sender.py

```diff
@@ -890,16 +950,117 @@ def send_stats(self, record: "Record") -> None:
         self._fs.push(filenames.EVENTS_FNAME, json.dumps(row))
         # TODO(jhr): check fs.push results?
 
+    def _output_raw_finish(self) -> None:
+        for stream, output_raw in self._output_raw_streams.items():
+            output_raw._stopped.set()
+
+            # shut down threads
+            output_raw._writer_thr.join(timeout=5)
+            if output_raw._writer_thr.is_alive():
+                logger.info("processing output...")
```

Comment:
can we have this one more informative?

Reply:
this is almost directly from the old code. it means we are processing the output stream during shutdown.   in the old code it was in the user process so it was an attempt to let the user know that something was happening.   I moved it to logger and kept it the same so it would be consistent between the two paths.